web/packages/teleport/src/lib/tdp/codec.ts

```diff
@@ -523,19 +529,38 @@ export default class Codec {
     return buffer;
   }
 
+  static keyboardButtonMessageLength =
```

Comment:
Is there any reason to define this as a class field?
I think the more common practice is to put it as a constant on top of the file.
```ts
const KEYBOARD_BUTTON_MESSAGE_LENGTH = ...
```

BTW I would do the same for `_keyScancodes`, it is a readonly value.